### ISSUE TITLE
Where to open PR

### ISSUE THREAD

**Issue by Iv-84** [hacash/fullnode#1]
Dear repo admins,
I'm working on a new feature for hacash. Should I keep on hacash/rust or move to hacash/fullnode?

**Comment by jojoin** [comment]
Yes, there is, the hacash/fullnode repo.

**Comment by yita0101** [comment]
> Dear repo admins, I'm working on a new feature for hacash. Should I keep on hacash/rust or move to hacash/fullnode?

Maybe this project will take less time to compile, I've been aware of that for a long time

**Comment by jojoin** [comment]
Yes, this is a significant refactoring in the Rust version, with clear module divisions that facilitate collaboration among engineers and greatly reduce compilation time. Future development will be based on this repository.
